### ISSUE TITLE
Wallet QT freezes up

### ISSUE THREAD

**Issue by ghost** [SyndicateLabs/SyndicateQT#5]
Wallet locks up repeatedly every so often.

**Comment by BurningMan44** [comment]
Wallet fix waiting for merge into synx code.

**Comment by Carvalh0XYZ** [comment]
What about this ? [https://vgy.me/4y6ZuA.png](url) What am I missing to sync ?

**Comment by paniccode** [comment]
Wallet freeze constantly, windows 10 x64

**Comment by H0neyP0ney** [comment]
Yep it's insane to handle coins on this wallet, unfortunately...
